packages/zeplin-extension-style-kit/elements/layer.js

```diff
@@ -106,10 +115,75 @@ class Layer {
         return declarations;
     }
 
+    blendBorders(accumulator, current) {
+        if (accumulator.fill.type === Fill.TYPE.COLOR && current.fill.type === Fill.TYPE.COLOR) {
+            return {
+                position: accumulator.position,
+                thickness: accumulator.thickness,
+                fill: {
+                    type: Fill.TYPE.COLOR,
+                    color: current.fill.color.blend(accumulator.fill.color)
+                }
+            };
+        }
+
+        if (accumulator.fill.type === Fill.TYPE.GRADIENT) {
+            return {
+                position: accumulator.position,
+                thickness: accumulator.thickness,
+                fill: Object.assign(
+                    {},
+                    accumulator.fill,
+                    {
+                        gradient: Object.assign(
+                            {},
+                            accumulator.fill.gradient,
+                            {
+                                colorStops: accumulator.fill.gradient.colorStops.map(({ position, color }) => ({
+                                    position,
+                                    color: current.fill.color.blend(color)
+                                }))
+                            }
+                        )
+                    }
+                )
+            };
+        }
+
+        return {
+            position: accumulator.position,
+            thickness: accumulator.thickness,
+            fill: Object.assign(
+                {},
+                current.fill,
+                {
+                    gradient: Object.assign(
+                        {},
+                        current.fill.gradient,
+                        {
+                            colorStops: current.fill.gradient.colorStops.map(({ position, color }) => ({
+                                position,
+                                color: color.blend(accumulator.fill.color)
+                            }))
+                        }
+                    )
+                }
+            )
+        };
+    }
+
     get elementBorder() {
         const { object: { borders } } = this;
 
-        return borders.length ? borders[borders.length - 1] : null;
+        if (!this.shouldDrawBorder()) {
+            return null;
+        }
+
+        if (borders.length === 1) {
+            return borders[borders.length - 1];
```

Comment:
if `borders.length === 1` then this is `borders[0]`? 😅

Reply:
```suggestion
            return borders[0];
```